train_model.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import h5py
+import numpy as np
+
+from keras.models import Sequential
+from keras.layers import Flatten, Dense, Lambda, Conv2D, Cropping2D, Dropout, MaxPooling2D
+from keras.optimizers import Adam
+from sklearn.model_selection import train_test_split
+
+from utils.resnet import resnet50_model
+from keras.backend import tensorflow_backend as backend
+

```

Comment:
[MUST]
リサイズ（画像の前処理）のためのライブラリ読み込み

```suggestion
import cv2
```